server/views/grid/nesting/nesting.config.json

```diff
@@ -1,3 +1,3 @@
 {
-    "order": 5
+    "order": 7
 }
```

Comment:
Fractal allows for arbitrary ordering of items in the sidebar (defaults to alpha otherwise).